pkg/auth/providerrefresh/daemon.go

```diff
@@ -105,7 +106,7 @@ func ParseCron(setting string) (cron.Schedule, error) {
 	}
 	schedule, err := cron.ParseStandard(setting)
 	if err != nil {
-		return nil, fmt.Errorf("Error parsing auth refresh cron: %v", err)
+		return nil, fmt.Errorf("error parsing auth refresh cron: %v", err)
```

Comment:
Could we include the string that we failed to parse? Or does the error from `cron.ParseStandard` include this?

(This hints at a missing test?)